pages/index.vue

```diff
@@ -253,7 +257,7 @@
           alt="OneMarkdown"
         />
         <div class="prose">
-          <h2>OneMarkdown will be a custom flavor of Markdown that supports all HTML tags</h2>
+            <h2>OneMarkdown will be a Markdown flavor that supports more than other flavors</h2>
```

Comment:
sort of sounds wrong

Reply:
eh

what do you suggest?

Reply:
@RedGuy12

```suggestion
            <h2>OneMarkdown will be a Markdown flavor that has a focus on many useful features</h2>
```

idk this?